omnibus/files/private-chef-cookbooks/private-chef/recipes/cleanup.rb

```diff
@@ -44,3 +44,20 @@
   ignore_failure true
 end
 
+#
+# Opscode Expander Reindexer from Chef Server 12.3.1 and earlier
+#
+
+# The reindexer has been removed since we no longer use RabbitMQ.
```

Comment:
Minor nit, this isn't why reindexer was removed. See the commit message in 56c156bc71201dc8bf921ef69cfff4db3e9ff898 for the reasoning.